### PR TITLE
Translate Contribution Guideline - Policy OSS

### LINKED ISSUE #27: Translate "Contribution Guidelines (Policies: Open source)" into Korean
link: https://azure.github.io/azure-sdk-korean/policies_opensource.html

### PR DESCRIPTION
**Pull request by ianychoi**
Corresponding to https://azure.github.io/azure-sdk/policies_opensource.html

Closes #27

### CODE REVIEW

File: docs/policies/opensource.md [hunk 1]
```diff
@@ -1,74 +1,73 @@
...
-* You can also capture the information about which people have to be notified when issues are filed for a service. To do that, you have to add the `# ServiceLabel: ` followed by the `%Label` that have to be applied to an issue in order for the people specified in the path below to be mentioned by the bot. 
-* If the code for a service is not inside the repo, you can use this special commented out path to allow issues to be tagged for a service: `#/<NotInRepo>/`
-* For the labels that are used with the `%` character, you can use spaces. Labels are delimited by the start of the `%` character.
+* `/.github/CODEOWNERS` 파일을 사용합니다.
+* `/sdk/<service name>/` (슬래시 기호를 시작과 끝 부분에 함께 사용) 통용 규칙을 따라하여 서비스 소유자를 정의합니다.
+  * 이 포맷을 사용하는 경우에는, 서비스 소유자들은 자동으로 빌드 알림 실패 경고를 구독하게 될 것입니다
+* GitHub가 마지막에 일치하는 표현 (규칙)을 사용하므로, 보다 일반적인 규칙을 파일 위쪽에 배치하고, 보다 구체적인 규칙은 파일 아래 부분에 배치합니다.
+* 소유자에 대해 GitHub에서 (지정하는) 사람에 대한 별칭만을 사용합니다 (예: `@person`). 내부 사용자, 내부 그룹 별칭 및 이메일 주소와 연결되지 않은 GitHub 그룹 및 GitHub 사용자는 현재 지원하지 않습니다.
```
Comment (i-hope9) [inline docs/policies/opensource.md:67]: 번역 누락된 부분 추가
```suggestion
* 소유자에 대해 GitHub에서 (지정하는) 사람에 대한 별칭만을 사용합니다 (예: `@person`). 내부 사용자, 내부 그룹 별칭 및 이메일 주소와 연결되지 않은 GitHub 그룹 및 GitHub 사용자는 현재 지원하지 않습니다.
* 해당 폴더의 PR에 자동 라벨링을 지정하고 싶다면,  `# PRLabel: ` 다음에 적용하려는 `%Label`의 내용이 있는 경로와 함께 항목 위에 주석 줄을 추가해야 합니다. 참고: 현재 와일드카드는 지원되지 않으며, 폴더당 하나의 라벨만 작동합니다. 
```

**Review by i-hope9 — CHANGES_REQUESTED** [review]
Reviewed docs/policies/opensource.md

File: docs/policies/opensource.md [hunk 2]
```diff
@@ -1,74 +1,73 @@
...
 
-{% include requirement/SHOULD %} develop in the open on GitHub. Seek feedback from the community on design choices and be active in conversations with the community.
+{% include requirement/SHOULD %} GitHub에서 오픈되어 있는 상태에서 개발을 진행합니다. 디자인 선택에 대한 부분을 커뮤니티에서 피드백을 찾아보며, 커뮤니티에서는 대화에 적극적으로 참여합니다.
 
-{% include requirement/MUST %} remain active in GitHub. Your client library is your primary touchpoint with the developer community, so it's important to keep up with the activity there. Issues and pull requests on GitHub must have an authoritative comment within one week of filing.
+{% include requirement/MUST %} GitHub에서 적극적으로 임해주세요. 여러분의 클라이언트 라이브러리는 개발자 커뮤니티에서 주된 접점이 되므로, 활동을 계속 이어가는 것이 중요합니다. 이슈와 풀 리퀘스트는 제출이 이루어지고 나서 1주일 이내에 신뢰할 수 있는 코멘트가 있어야 합니다.
 
-{% include requirement/MUST %} review the Microsoft Open Source Guidelines' [community section](https://docs.opensource.microsoft.com/releasing/foster-your-community.html) for more information on fostering a healthy open-source community.
+{% include requirement/MUST %} 건강한 오픈 소스 커뮤니티를 지향하기 위한 보다 자세한 정보에 대해서는 Microsoft Open Source Guidelines에 있는 [커뮤니티 섹션](https://docs.opensource.microsoft.com/releasing/foster-your-community.html)을 살펴 봅니다.
```
Comment (kypark7) [inline docs/policies/opensource.md:19]:
```suggestion
{% include requirement/MUST %} 건강한 오픈 소스 커뮤니티를 지향하기 위한 보다 자세한 정보에 대해서는 Microsoft Open Source Guidelines에 있는 [커뮤니티 섹션](https://docs.opensource.microsoft.com/releasing/foster-your-community.html)을 살펴봅니다.
```
no space needed

File: docs/policies/opensource.md [hunk 3]
```diff
@@ -1,74 +1,73 @@
...
 
-{% include requirement/MUST %} use the [Microsoft CLA](https://cla.opensource.microsoft.com/). Microsoft makes significant contributions to [cla-assistant](https://cla-assistant.io/). It is the easiest way to ensure the CLA is signed by all contributors.
+{% include requirement/MUST %} [Microsoft CLA](https://cla.opensource.microsoft.com/)를 사용합니다. Microsoft에서는 [cla-assistant](https://cla-assistant.io/)에 대해 상당한 기여를 하고 있습니다. 이는 모든 컨트리뷰터들이 CLA를 서명하였는지 확인하는 가장 쉬운 방법입니다.
 
-{% include requirement/MUST %} include a copyright header at the top of every source file (including samples). See the [Microsoft Open Source Guidelines](https://docs.opensource.microsoft.com/releasing/copyright-headers.html) for example headers in various languages.
+{% include requirement/MUST %} 모든 소스 파일 (샘플 포함) 상단에 저작권(copyright) 헤더를 포함합니다. 다양한 언어에 대한 헤더 예제로 [Microsoft Open Source Guidelines](https://docs.opensource.microsoft.com/releasing/copyright-headers.html)을 살펴 봅니다.
```
Comment (kypark7) [inline docs/policies/opensource.md:23]:
```suggestion
{% include requirement/MUST %} 모든 소스 파일 (샘플 포함) 상단에 저작권(copyright) 헤더를 포함합니다. 다양한 언어에 대한 헤더 예제로 [Microsoft Open Source Guidelines](https://docs.opensource.microsoft.com/releasing/copyright-headers.html)을 살펴봅니다.
```
no space needed

**Comment by ianychoi** [comment]
Thank you for the reviews! Now waiting for @yeongseon - plz review and feel free to merge if there are no further issues.

**Pull request merged**